helix-core/src/main/java/org/apache/helix/task/AbstractTaskDispatcher.java

```diff
@@ -192,11 +191,8 @@ public void updatePreviousAssignedTasksStatus(
             paMap.put(pId, new PartitionAssignment(instance, requestedState.name()));
           }
           assignedPartitions.get(instance).add(pId);
-          if (LOG.isDebugEnabled()) {
```

Comment:
If we get rid of isDebugEnabled check. Shall we refactor this for the code base? This is not task frame work only behavior.

Reply:
I do think so. Since this check is no longer needed after using slf4j.Logger.

Reply:
Then let's either have a different PR for that. Or let's create a ticket to track that.